crates/sui-bridge-cli/src/lib.rs

```diff
@@ -90,6 +90,12 @@ pub enum BridgeCommand {
         #[clap(long = "eth-rpc-url")]
         eth_rpc_url: String,
     },
+    /// Print current registration info
+    #[clap(name = "print-bridge-registration-info")]
+    PrintBridgeRegistrationInfo {
+        #[clap(long = "sui-rpc-url")]
```

Comment:
I know that you know that I know that you know... but you can just say `long` when the long name matches the field name. Though maybe was on purpose...

Reply:
will fix this in a later PR